src/player.cpp

```diff
@@ -141,6 +145,1107 @@ void Player::clearHud()
 	}
 }
 
+// FROM ClientEnvironment::step
```

Comment:
these comments aren't very useful

Reply:
They're supposed to help me find things after rebasing again, but this seems to be less of a problem now, so I'll hunt and remove these.

Reply:
Removed (I hope).